plugins/scaffolder-backend/api-report.md

```diff
@@ -428,8 +428,18 @@ export class DatabaseTaskStore implements TaskStore {
   // (undocumented)
   heartbeatTask(taskId: string): Promise<void>;
   // (undocumented)
-  list(options: { createdBy?: string; status?: TaskStatus_2 }): Promise<{
```

Comment:
I think it's best to leave these options as required, otherwise it's a breaking change to the API for implementers of this API. It's a mistake that it's required for sure, but let's not break it just yet.

Reply:
Ok, will fix